CRITICAL - 2022-03-13 11:57:53 --> file_get_contents(http://localhost:8080/api/pagamento): Failed to open stream: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp2\\htdoc...', 13)
#1 C:\xampp2\htdocs\contratos\app\Controllers\Home.php(13): file_get_contents('http://localhos...')
#2 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(825): App\Controllers\Home->index()
#3 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp2\htdocs\contratos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
